lib/galaxy/managers/history_contents.py

```diff
@@ -631,9 +631,10 @@ def parse_type_id_list(self, type_id_list_string, sep=","):
         return [self.decode_type_id(type_id) for type_id in type_id_list_string.split(sep)]
 
     def _add_parsers(self):
+        database_connection: str = self.app.config.database_connection
         super()._add_parsers()
```

Comment:
```suggestion
        super()._add_parsers()
        database_connection: str = self.app.config.database_connection
```
consider calling `super()` first: that way it cannot be implied that the parent implementation somehow depends on that assignment statement.

Reply:
Done, makes sense, thanks!